src/CreatePlayerButton.js

```diff
@@ -0,0 +1,35 @@
+import { supabase } from "./supabase_client";
+import { useEffect, useState } from "react";
+import JoinRoomButton from "./JoinRoomButton";
+
+const CreatePlayerButton = (props) => {
+  const [hasUserInput, setHasUserInput] = useState();
+  const { player, setPlayer, setPage } = props;
+  const onChange = (e) => {
+    setHasUserInput(e.target.value);
+  };
+  const handleClick = async () => {
+    setPlayer(hasUserInput);
+    const username = player;
+    console.log(username);
+    let { error, status } = await supabase
```

Comment:
default to `const` unless you plan on modifying the variable (which most times you shouldn't)